scripts/refresh-after-snapshot.sh

```
@@ -206,6 +218,25 @@ refresh_cdi_certificates() {
+refresh_metallb_certificates() {
+    # After recert, the MetalLB webhook server still has certs signed by the old
+    # CA. Delete the OLM-managed cert secret so it gets regenerated, then restart
+    # the webhook server pod to pick up the new cert.
+    if ! oc get crd ipaddresspools.metallb.io &>/dev/null; then
+        return 0
+    fi
+    echo "  Refreshing MetalLB webhook certificates..."
+    oc delete secret metallb-operator-webhook-server-cert -n metallb-system --ignore-not-found
+    oc delete pod -n metallb-system -l control-plane=controller-manager --ignore-not-found 2>/dev/null || true
+    oc delete pod -n metallb-system -l component=webhook-server --ignore-not-found 2>/dev/null || true
+    retry_until 120 5 '[[ "$(oc get deploy metallb-operator-webhook-server -n metallb-system -o jsonpath='"'"'{.status.readyReplicas}'"'"' 2>/dev/null)" == "1" ]]' || {
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)